scripts/triangles.js

```diff
@@ -0,0 +1,9 @@
+function isTriangle(length1, length2, length3) {
+  if ((length1 + length2 > length3) && (length2 + length3 > length1) && (length1 + length3 > length2)) {
```

Comment:
this is pretty lengthy. could we extract this logic (checking the lengths) out to another function?